packages/xo-server/src/xapi/index.js

```diff
@@ -247,9 +247,10 @@ export default class Xapi extends XapiBase {
     )::ignoreErrors()
   }
 
-  async setHostProperties(id, { nameLabel, nameDescription }) {
+  async setHostProperties(id, { iscsiIqn, nameLabel, nameDescription }) {
     const host = this.getObject(id)
     await Promise.all([
+      iscsiIqn != null && host.update_other_config('iscsi_iqn', iscsiIqn),
```

Comment:
`iscsiIqn` cannot be null.

Reply:
`iscsiIqn` can be null if user clear it from the input.